cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -2839,6 +2945,138 @@ fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(
             }
         }
 
+        Opcode::Splat => {
+            let ty = ty.unwrap();
+            assert_eq!(ty.bits(), 128);
+            let src_ty = ctx.input_ty(insn, 0);
+            assert!(src_ty.bits() < 128);
+            let src = input_to_reg_mem(ctx, inputs[0]);
+            let dst = get_output_reg(ctx, outputs[0]);
+
+            fn emit_insert_lane<C: LowerCtx<I = Inst>>(
+                ctx: &mut C,
+                src: RegMem,
+                dst: Writable<Reg>,
+                lane: u8,
+                ty: Type,
+            ) {
+                if !ty.is_float() {
+                    let (sse_op, w_bit) = match ty.lane_bits() {
```

Comment:
What is `w_bit` (is it `is64` as elsewhere)?

Reply:
REX.W... changed.